pyproject.toml

```diff
@@ -69,6 +69,7 @@ markers = [
 ]
 
 [tool.mypy]
+python_version = "3.8"
```

Comment:
can you move this adjacent to `platform = "linux-64"`

The config is ordered the same as the mypy docs.

Reply:
moved